docs/content/index.md

```diff
@@ -2,60 +2,153 @@
+* **Dimension columns**: Here we have four dimensions of publisher, advertiser, gender, and country.
+They each represent an axis of the data that we’ve chosen to slice across.
 
-Druid is a system that you can set up in your organization next to Hadoop. It provides the ability to access your data in an interactive slice-and-dice fashion. It trades off some query flexibility and takes over the storage format in order to provide the speed.
+* **Metric columns**: These are impressions, clicks and revenue. These represent values, usually numeric,
+which are derived from an aggregation operation – such as count, sum, and mean (we also run variance and higher moment calculations).
+Also known as measures in standard OLAP terminology.
 
-We have more details about the general design of the system and why you might want to use it in our [White Paper](http://static.druid.io/docs/druid.pdf) or in our [Design](Design.html) doc.
+Individually, the events are not very interesting, however, summarizations of this type of data can yield many useful insights.
+Druid can optionally choose to summarize this raw data at ingestion time using a process we refer to as "roll-up".
+Roll-up is a first-level aggregation operation over a selected set of dimensions, equivalent to (in pseudocode):
 
+    GROUP BY timestamp, publisher, advertiser, gender, country
+      :: impressions = COUNT(1),  clicks = SUM(click),  revenue = SUM(price)
 
-When Druid?
-----------
+The compacted version of our original raw data looks something like this:
 
-* You need to do interactive aggregations and fast exploration on large amounts of data
-* You need ad-hoc analytic queries (not a key-value store)
-* You have a lot of data (10s of billions of events added per day, 10s of TB of data added per day)
-* You want to do your analysis on data as it’s happening (in real-time)
-* You need a data store that is always available with no single point of failure.
+     timestamp             publisher          advertiser  gender country impressions clicks revenue
+     2011-01-01T01:00:00Z  ultratrimfast.com  google.com  Male   USA     1800        25     15.70
+     2011-01-01T01:00:00Z  bieberfever.com    google.com  Male   USA     2912        42     29.18
+     2011-01-01T02:00:00Z  ultratrimfast.com  google.com  Male   UK      1953        17     17.31
+     2011-01-01T02:00:00Z  bieberfever.com    google.com  Male   UK      3194        170    34.01
 
-Architecture Overview
----------------------
+In practice, we see that rolling up data can dramatically reduce the size of data that needs to be stored (often by a factor of 100).
+This storage reduction does come at a cost; as we roll up data, we lose the ability to query individual events. Phrased another way,
+the rollup granularity is the minimum granularity you will be able to query data at. Hence, Druid ingestion specs define this granularity as the `queryGranularity` of the data.
```

Comment:
I'm not sure if this would make things clearer or more confusing, but we could tell people that Druid treats queryGranularity-truncated time like a dimension. It's a very special dimension, but it still behaves like one in the olap sense.